app/models/mixins/miq_web_server_runner_mixin.rb

```diff
@@ -14,4 +14,32 @@ def do_before_work_loop
     # Since puma/thin traps interrupts, log that we're going away and update our worker row
     at_exit { do_exit("Exit request received.") }
   end
+
+  module ClassMethods
+    def start_worker(*args)
+      worker = self.new(*args)
```

Comment:
TODO: Rename worker to runner here since @matthewd cleverly renamed `self to a `Runner` so it now makes sense